<?xml version="1.0"?>

<launch>
  <arg name="frame"/>
  <arg name="worldFrame" default="world"/>

  <node name="controller" pkg="crazyflie_controller" type="acados_nmpc" output="screen">
    <param name="frame" value="$(arg frame)" />
    <param name="worldFrame" value="$(arg worldFrame)" />
  </node>
</launch>
